from urllib.parse import urlparse
from django.core.exceptions import ValidationError
